mypyc/test-data/analysis.test

```diff
@@ -10,23 +10,17 @@ def f(a: int) -> None:
 [out]
 def f(a):
     a :: int
-    r0 :: short_int
     x :: int
     r1 :: bool
-    r2, r3, r4 :: native_int
+    r3 :: native_int
```

Comment:
Unfortunately, this leaves gaps in the register numbering. A relatively easy way to fix this would be to give `LoadInt` values different names with independent numbering, such as `i0`, `i1`, etc., instead of `r0`, `r1`, and so on. This way we'd just filter out all `iX` names, and `rX` would remain without gaps.